analyses/org.osate.analysis.model/src-gen/org/osate/analysis/model/analysis/AnalysisElement.java

```diff
@@ -0,0 +1,48 @@
+/**
+ */
+package org.osate.analysis.model.analysis;
```

Comment:
Why not simply package org.analysis.model?

Reply:
I don't mind the shorter `org.analysis.model`, but everything else starts with `org.osate`

Reply:
Changed the base package for the Analysis ECore package to be `org.osate.model`.  Regenerated all the model classes.